tests/tests/control_stochy

```diff
@@ -33,11 +33,12 @@ export IAER=5111
 export DO_SPPT=.T.
 export DO_SHUM=.T.
 export DO_SKEB=.T.
+export RESTART_INTERVAL=6
 export SKEB=0.3
 export SHUM=0.003
 export SPPT=0.2
 
 
-export FV3_RUN=control_run.IN
+export FV3_RUN=control_stochy_run.IN
```

Comment:
Does this run script exist? I don't see it in the current develop/tests/fv3_conf directory or in your PR branch.

Reply:
hmm, maybe I didn't add it. Let me check.